gitlab-ci/common.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Black Duck Software, Inc. All rights reserved worldwide.
+variables:
+    ARTIFACTORY_URL: https://artifactory.tools.duckutil.net/artifactory
+    ARTIFACTORY_REGISTRY: artifactory.tools.duckutil.net:5002
+    ARTIFACTORY_JFROG_URL: $ARTIFACTORY_URL/sigma-local/devops/helpers/jfrog/jfrog.linux_x86_64
+
+.jfrog_before_script:
+    - curl --silent --output jfrog $ARTIFACTORY_JFROG_URL
+    - chmod a+x jfrog
+    - ./jfrog rt config import ${SIGMA_CICD_ARTIFACTORY_CONFIG}
+
+# Common Rules
+.manually_run_in_merge_requests:
+    if: $CI_MERGE_REQUEST_IID && $CI_MERGE_REQUEST_EVENT_TYPE != 'merge_train'
+    when: manual
+.never_run_for_scheduled_jobs:
+    if: $CI_PIPELINE_SOURCE == "schedule"
+    when: never
+.only_run_for_scheduled_jobs:
+    if: $CI_PIPELINE_SOURCE != "schedule"
+    when: never
```

Comment:
Is this right? Should be always?

Reply:
Not sure if we should have this or not, or if the rule should look something like:
```
    rules:
        - if: '$CI_PIPELINE_SOURCE == "schedule" && $SCHEDULE_TAG == "publish_hybrid_fe_artifacts"'
```

This is an example from some scheduled job in Sigma, we could use schedule tag of `pop_sca_scan` or something.